### ISSUE TITLE
Possibility to deactivate marketplace

### ISSUE THREAD

**Issue by the-other-dev** [directus/directus#22582]
### Describe the Improvement

Is it possible to add a configuration option to deactivate the marketplace?

Deactivating includes the following:
* Marketplace menu item is not displayed in the settings menu

Background

In self-hosted situations you might not be able to use the (public) marketplace due to compliance reasons.

Workarounds

a) Configure your own marketplace and implement the API to serve an empty marketplace.
b) Deny access (e.g. via network policies) to the public marketplace and receive a timeout error/internal error in the UI when clicking on the marketplace menu item (slightly bad user experience)

**Comment by achille2k** [comment]
I agree

**Comment by YannickArmspach** [comment]
Agreed. In the meantime, you can use the variable `EXTENSIONS_LIMIT=0` to disable the install
